### DISCUSSION TITLE
[Feature request] Link opens in - Replace the current tab

### DISCUSSION THREAD

**Discussion by TrueAccelerator** [linuxserver/Heimdall#680] [Ideas]
At the moment, the pages opened in the same tab are treated as embedded. This makes certain sites (gmail) impossible to use, while others suffer from mozzilla's policy which makes it impossible to use addons for such sites. This deactivates addons like mouse gestures. 

It is, of course possible to just make Heimdall open links in new tabs, but this increases the number of necessary clicks (to close heimdall instances once they are no longer needed) and makes it difficult to navigate in tab tree view (heimdall pages are treated as blank new tabs)

Suggestion: Add a new choice to 'link open in' that either replaces the current tab or, if that's not possible, opens it in a new tab and immediately collapses the heimdall page used to open it.

**Comment by j0nnymoe** [comment]
The feature is already in settings: 
![9594](https://user-images.githubusercontent.com/10374553/96362961-12b33d80-1129-11eb-9e45-5cbaa0a698a5.png)
Unless I'm missing something?

**Comment by TrueAccelerator** [comment]
the thing is, the heimdall tab remains active, so for users of tree tab add ons it's rather inconvenient. I guess I could just leave one heimdall instance all the time and make all the sites stem from it but it's not ideal either.

What I'd love to see is a setting that opens a site in a new tab and closes the heimdall instance used to launch said site

**Comment by seraasch** [comment]
Is this related to how apps marked as "open in a new tab" all open in the *same* tab (i.e. you can't open two of these apps simultaneously)?  I'm seeing that opening a second app, the second replaces the first instead of opening another new tab...

I can see the value of the existing behavior, but I'd like to have the option to open a new tab every time

**Comment by KodeStar** [comment]
Is this fixed with rel="noopener noreferrer" added to links now?
